lib/client/api.go

```diff
@@ -1144,6 +1145,7 @@ func (c *Config) SaveProfile(dir string, makeCurrent bool) error {
 	cp.ForwardedPorts = c.LocalForwardPorts.String()
 	cp.SiteName = c.SiteName
 	cp.TLSRoutingEnabled = c.TLSRoutingEnabled
+	cp.AuthConnector = c.AuthConnector
```

Comment:
Could you add a simple SaveProfile/LoadProfile test?

Reply:
I have extended `TestProfileBasics` with it.

Reply:
Missing push?

Reply:
https://github.com/gravitational/teleport/blob/1aed847eb5b79ea069fd2f1ca3ffdce4a377b4d1/api/profile/profile_test.go#L47
it's there, just 1 line because it check if save and load works.

Reply:
My bad!